include/verilated_timing.cpp

```diff
@@ -144,6 +144,66 @@ void VlTriggerScheduler::dump(const char* eventDescription) {
 }
 #endif
 
+//======================================================================
+// VlDynamicTriggerScheduler:: Methods
+
+bool VlDynamicTriggerScheduler::evaluate() {
+#ifdef VL_DEBUG
+    dump();
+#endif
+    std::swap(m_suspended, m_evaluated);
+    for (auto& coro : m_evaluated) coro.resume();
+    m_evaluated.clear();
+    return !m_triggered.empty();
+}
+
+void VlDynamicTriggerScheduler::doPostUpdates() {
+#ifdef VL_DEBUG
+    if (!m_post.empty()) {
+        VL_DEBUG_IF(VL_DBG_MSGF("         Doing post updates for processes:\n");  //
+                    for (auto& susp
+                         : m_post) {
```

Comment:
```suggestion
                    for (auto& susp : m_post) {
```

And maybe const auto&.  Likewise elsewhere.

Reply:
The weird line break is not my style choice :) it's `clang-format`'s fault.

Reply:
As for the const: https://github.com/verilator/verilator/pull/3653